docs/conf.py

```diff
@@ -0,0 +1,78 @@
+import configparser
+from datetime import date
+import os
+import subprocess
+
+
+config = configparser.ConfigParser()
+config.read(os.path.join('..', 'setup.cfg'))
+
+# Project -----------------------------------------------------------------
+author = config['metadata']['author']
+copyright = f'2020-{date.today().year} audEERING GmbH'
+project = config['metadata']['name']
+# The x.y.z version read from tags
+try:
+    version = subprocess.check_output(
+        ['git', 'describe', '--tags', '--always']
+    )
+    version = version.decode().strip()
+except Exception:
+    version = '<unknown>'
+title = f'{project} Documentation'
+
+
+# General -----------------------------------------------------------------
+master_doc = 'index'
+extensions = []
+source_suffix = '.rst'
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+pygments_style = None
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.napoleon',  # support for Google-style docstrings
+    'sphinx.ext.viewcode',
+    'sphinx.ext.intersphinx',
+    'sphinx_autodoc_typehints',
+    'sphinx_copybutton',  # for "copy to clipboard" buttons
+    'matplotlib.sphinxext.plot_directive',  # include resulting figures in doc
+]
+
+# Do not copy prompot output
+copybutton_prompt_text = r'>>> |\.\.\. '
+copybutton_prompt_is_regexp = True
+
+# Mapping to external documentation
+intersphinx_mapping = {
+    'audmetric': ('https://audeering.github.io/audmetric/', None),
+    'pandas': ('https://pandas.pydata.org/pandas-docs/stable/', None),
+    'python': ('https://docs.python.org/3/', None),
+}
+
+# Disable Gitlab as we need to sign in
+linkcheck_ignore = [
+    'https://gitlab.audeering.com',
+]
+
+# Plotting
+plot_include_source = True
+plot_html_show_source_link = False
+plot_html_show_formats = False
+plot_pre_code = ''
+plot_rcparams = {
+    # 'figure.figsize': '8, 4.5',  # inch
```

Comment:
do we need this comment?

Reply:
No, I just realize that I have not finished this pull request, see also the `distplot` warning ;)

Will do and come back here.

Reply:
Unfortunately, I'm struggling with using `histplot` instead of `distplot` as it does not update the color and produces in general quite a different result:

![image](https://user-images.githubusercontent.com/173624/122524142-1381d380-d018-11eb-8ca7-7b2a1d0742d4.png)

Any advice how to solve this, or should we do it in another pull request as well?

Reply:
I removed the `plot_rcparams` setting from the config file.

Reply:
Besides the deprecation warning for `distplot` this should be finished now.

Reply:
Ok, we decided to work on the deprecation warning in a separate PR, see https://github.com/audeering/audplot/issues/2